pkg/onit/cluster/simulator.go

```diff
@@ -359,6 +361,15 @@ func (s *Simulator) createPod() error {
 							Name:  gnmiInsecurePortEnv,
 							Value: fmt.Sprintf("%d", s.Port()),
 						},
+						{
+							Name: kube.NamespaceEnv,
```

Comment:
It looks like you defined namespaceEnv as constant but you used something else. Am I missing something?

Reply:
That's the name of the environment variable (`NAMESPACE`), and the value is taken from the k8s configuration

Reply:
Sorry I thought you meant the variable used here :-)

Reply:
yeah - i had defined it, and then saw it had already been defined, and forgot to remove it. Removed now.